### PR TITLE
feat: policy service beta

### PR DESCRIPTION
**Pull request by gkc**
**- What I did**
- new BETA policy binary, npp_atserver: fetches its rules from atServer as persistent store
- new BETA binary, np_admin: exposes an API via http for setting policy rules & seeing policy info, events etc
- new BETA webapp bundled with np_admin to provide a simple illustrative UI

Note that this is still only alpha quality but the intention is that will be beta quality in our 5.7.0 release

**- How I did it**
See commits

**- How to verify it**
- Tests pass
- multibuild and dockerfile.package work as intended

**TODO in subsequent PRs**
- take up an [at_client.AtRpc enhancement](https://github.com/atsign-foundation/at_client_sdk/pull/1388)
- add concrete types for events, logs etc rather than json/map nonsense
- add permitOpen to the NPAAuthCheckRequest so we can have policy be able to know if a request should be denied when no matching permitOpen found
- refactor: rename all the NPA/npa classes/files/directories to NPP/npp
- UI refactoring - split into pieces, reduce duplicate code
- UI<->API enhancements re events, logs etc
- Documentation
- Postgres impl

### CODE REVIEW

**Comment by xavierchanth** [comment]
Started a multibuild here: https://github.com/atsign-foundation/noports/actions/runs/10723158846

File: tools/multibuild/Dockerfile.package [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  dart pub get --enforce-lockfile; \
+  dart run build_runner build --delete-conflicting-outputs; \
+  dart compile exe bin/activate_cli.dart -v -o /sshnp/at_activate; \
+  dart compile exe bin/sshnp.dart -v -o /sshnp/sshnp; \
+  dart compile exe bin/npt.dart -v -o /sshnp/npt; \
+  dart compile exe bin/npp_file.dart -v -o /sshnp/npp_file; \
+  dart compile exe bin/npp_atserver.dart -v -o /sshnp/npp_atserver; \
+  dart compile exe bin/sshnpd.dart -v -o /sshnp/sshnpd; \
+  dart compile exe bin/srv.dart -v -o /sshnp/srv; \
+  dart compile exe bin/npp.dart -v -o /sshnp/npp; \
```
Comment (xavierchanth) [inline tools/multibuild/Dockerfile.package]: This line needs to be removed, since it is now `npp_atserver` which is already there 3 lines above

Comment (gkc) [reply tools/multibuild/Dockerfile.package]: Done

Comment (gkc) [reply tools/multibuild/Dockerfile.package]: Running another multibuild [here](https://github.com/atsign-foundation/noports/actions/runs/10725814755)

Comment (gkc) [reply tools/multibuild/Dockerfile.package]: multibuild ran ok

**Review by xavierchanth — APPROVED** [review]

**Comment by xavierchanth** [comment]
Cutting a release (from trunk): https://github.com/atsign-foundation/noports/releases/tag/v5.7.0-alpha-5

**Pull request merged**